src/main/scala/cognite/spark/v1/EventsRelation.scala

```diff
@@ -101,23 +101,12 @@ class EventsRelation(config: RelationConfig)(val sqlContext: SQLContext)
   override def getFromRowAndCreate(rows: Seq[Row]): IO[Unit] = {
     val events = fromRowWithFilteredMetadata(rows)
 
-    client.events
-      .create(events)
-      .handleErrorWith {
-        case e: CdpApiException =>
-          if (e.code == 409) {
-            val existingExternalIds =
-              e.duplicated.get.map(j => j("externalId").get.asString.get)
-            resolveConflict(existingExternalIds, events)
-          } else { IO.raiseError(e) }
-      } *> IO.unit
-  }
-
-  def resolveConflict(existingExternalIds: Seq[String], events: Seq[EventCreate]): IO[Unit] =
     upsertAfterConflict[Event, EventUpdate, EventCreate, Events[IO]](
```

Comment:
I don't think the name `upsertAfterConflict` makes much sense anymore, since now seems to be the primary entrypoint for doing upserts?